kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -132,16 +175,33 @@
 
   /**
    * Create a {@link Dataset} for the given dataset or view URI.
+   * {@code create} returns an empty dataset. You can use 
+   * {@link DatasetWriter} to populate your dataset.
```

Comment:
This best practice is not followed in our current Javadocs, nor in Javadocs from Google, Oracle (Sun), VMWare, or others I checked. We can consider making this part of the standard after 1.0.

Removed the link to DatasetWriter.